frontend/src/reports/ReportProvider.tsx

```diff
@@ -208,10 +207,11 @@ function ReportProvider(props: ReportProviderProps) {
           Until then, in accordance with our{" "}
           <Link to={METHODOLOGY_TAB_LINK}>methodology</Link>, the following
           states appear grey when viewing COVID-19 maps featuring
-          hospitalizations and deaths: <b>Hawaii</b>, <b>Nebraska</b>,{" "}
-          <b>South Dakota</b>, and <b>Wyoming</b>. <b>Delaware</b> is included
-          when viewing hospitalizations, but not deaths, and <b>Rhode Island</b>{" "}
-          is included when viewing deaths, but not hospitalizations.
+          hospitalizations and deaths: <b>Hawaii</b>, <b>Missouri</b>,{" "}
+          <b>Nebraska</b>, <b>South Dakota</b>, and <b>Wyoming</b>.{" "}
+          <b>Delaware</b> is included when viewing hospitalizations, but not
+          deaths, and <b>Rhode Island</b> is included when viewing deaths, but
+          not hospitalizations.
```

Comment:
I want to rewrite this text, it's still confusing; I have some ideas for clarifying it but I'll do in a new PR since it's unrelated to this update

Reply:
I also dislike it